app/experimenter/nimbus-ui/src/components/PageResults/GraphsWeekly/index.tsx

```diff
@@ -104,8 +107,8 @@ const GraphsWeekly = ({
 
   const [open, setOpen] = useState(false);
   const [embedded, setEmbedded] = useState(false);
-  const graphsVisibleClass = !open ? "d-none" : "";
-  const graphsHiddenClass = open ? "d-none" : "";
+  const graphsVisibleClass = !open ? "d-none" : undefined;
+  const graphsHiddenClass = open ? "d-none" : undefined;
```

Comment:
Empty React classNames should be set to `undefined` so `className=""` isn't rendered on the element.

Reply:
Curious, can the `classnames` utility be used here? Like `const graphsHiddenClass = classNames(open && "d-none");`

Reply:
Probably, I saw this in a drive-by sort of way editing another part of the file and didn't necessarily want to tweak further but hey might as well since I already changed the line and you called it out here. 😎

Reply:
I did end up changing this, but it feels a tad awkward living in this PR since I no longer touch this file otherwise. Oh well. 😆